lms/services/region.py

```diff
@@ -0,0 +1,18 @@
+from lms.models import Region
+
+
+class RegionService:
+    def __init__(self, region_code: str, authority: str):
+        self._region_code = region_code
+        self._authority = authority
+
+    def get(self) -> Region:
+        """Return the region that the app is currently running in."""
+        return Region(code=self._region_code, authority=self._authority)
+
+
+def factory(_context, request) -> RegionService:
+    return RegionService(
+        region_code=request.registry.settings["region_code"],
+        authority=request.registry.settings["h_authority"],
+    )
```

Comment:
Getting the current region is a perfect fit for a service, with the service factory being able to pass in the necessary settings from thr `request`.

Having said that, this service is only used in one place, in `run_data_task.py`, if that file called the `get_current_region()` function from the models then this service wouldn't be needed.